### DISCUSSION TITLE
API to download grades?

### DISCUSSION THREAD

**Discussion by jeremiah-ang** [community/community#37415] [GitHub Education]
Is there any API to download student's grades? 

Currently, the only way that I am aware of is via the classroom website

![image](https://user-images.githubusercontent.com/17397188/198176543-ed8a7744-588a-4047-81aa-56c876c32357.png)

**Comment by ossanna16** [comment]
Hi there @jeremiah-ang and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Comment by samus-aran** [comment]
Hi @jeremiah-ang 👋 

Thank you for asking a great question. Currently the only way(s) to acquire students' grades are via the Classroom interface itself and downloading the CSV (which you have screenshot).
